Source/Private/common.ps1

```diff
@@ -63,6 +63,11 @@ function _callAPI {
       $params.Add('Uri', $Url)
       $params.Add('UserAgent', (_getUserAgent))
 
+      #always use utf8 and json as default content type instead of xml
+      if ($false -eq $PSBoundParameters.ContainsKey("ContentType")) {

```

Comment:
@DarqueWarrior can you check if you agree? I thing it is a good approach because
1. we use json by default anyways and never xml
2. UTF8 should always be used if not configured otherwise.

A next step could be to remove the content types in all the other cmdlet so this problem does not occur anymore. We had this now the third time.